### ISSUE TITLE
Add in-world crafting speed configs

### ISSUE THREAD

**Issue by hexaguin** [AppliedEnergistics/Applied-Energistics-2#322]
Right now, it would appear that crystal growth rates are fixed. It would be helpful to have a config option for growth speed, so that pack creators, server owners, and players can adjust the balance of crystal growth.

**Comment by s3ptum** [comment]
Here is my take on the issue. I agree with the point of balance. I am a server owner and am setting up one for the 4th time. I also want to say I loved the original AE. AE2 seems a bit more confusing but I'm willing to learn. From what I've seen on the net, you can make an accelerator with out the seeds. This means the mod authors already set up a way to circumvent the growth time to under a minuet. Going to the nether is a requirement for this also.

To me that's balance. BUT, in the interest of keeping resource usage low, there are plugins like Clear Lag that remove all items sitting on the ground every interval you choose. If this interval is shorter than any of the growth times then it essentially breaks the game. 

Even knowing this I've kept AE2 on the server during the server's alpha stage. I'm still testing TiC, Buildcraft, RailCraft, TE, and Big reactors interactions. By the way its a Cauldron server on Forge 1291. 

I would appreciate it if there was a server side companion jar that could allow us to force the config change. Letting us get an instant return on the seeds at a 1:1 or less. Hopefully we could keep the amount of server client checks down. Not sure how that works. I only do hardware repair not code.

P.S. Thanks for keeping this mod alive I didn't know the original author left. The public never understands the way it works behind the scenes doing the coding and such.

Sincerely,

s3ptum(Steve)

**Comment by bookerthegeek** [comment]
Just fyi, clear lag has a blacklist for entities not to clear. Easy enough to add it to it.

On the otherhand, infinity evolved expert mode increases the time to 16 hours. Maybe take a peek at the scripts in it and you should be able to figure out how they modified the time.

Tl;dr
This feature already exists within minetweaker support.

**Comment by s3ptum** [comment]
Interesting. I will take a look. Thanks for the tip.

**Comment by Mementh** [comment]
as a user having this option is imho a good thing.. 

currently people can make one of two designs..  A river system ( allows you to possibly see how its gorwing) or a single space with other mods to put in and remove when they are fully grown.

there is a addon but right now the version of AE2 that works with that latest version of the thaumcraft addon does not work with AE2Stuff which has a growth chamber. 


or making a way to make the accelerators use more power but give more bonus growth.



people like the option to have faster growth at complexity/cost. its a trade off of sorts. 

AE can't guarantee those other mods will be around or compatable EVER! so giving a bonus accleration with increased power usage give a balance.

I would say you can make it insane increase since people can set this to turn off when not crafting using a toggle buss and level emiters. so 1 acceleration card = 10x power usage but only 2x speed increase? 2 cards 100x power 3x growth. 3 cards 1000x power 4x growth

per https://www.reddit.com/r/feedthebeast/comments/3evuz0/how_long_to_grow_pure_certus_for_ae2/ 

1 accelerator down to 3 mins 8 seconds ish
4 accelerators down to 31 seconds
6 accelerators down to 14.5 seconds

imho giving a option to the admin for a speed acceleration or slowing down or using accelerations cards can add more complexity but make people happier with what they have
